### DISCUSSION TITLE
Downloader doesn't open

### DISCUSSION THREAD

**Discussion by luxiferia** [nianeyna/ao3downloader#89] [Q&A] [answered]
The downloader window doesn't stay open, I run the program but the window closes on its own immediately after opening. I have tried everything in the troubleshooting suggestions, I've also tried to uninstall and reinstall several python releases from the 3.9.0. Help?

**Comment by nianeyna** [comment]
If you've done all the troubleshooting steps I unfortunately don't have a lot of ideas. What OS are you on and what folder location are you running the script from?

**Reply by luxiferia** [reply]
Sorry for taking so long to answer. OS is Windows 10, folder user/documents/ao3-downloader. I tried again just now, and the same issue occurred. Which is weird, last time I used it december last year I didn't have any issues.
I finally managed to get a screenshot, took me ages bc this blinks in like less than a second, I had to run it from an external drive to get it, but the same happens if I try to run it from the user/documents/ao3-downloader folder
![image](https://github.com/nianeyna/ao3downloader/assets/134893423/5b19c20b-e33a-4988-824b-edcf22751ba0)

**Reply by luxiferia** [reply, marked as answer]
OK I managed to make it work in a different desktop from the usb stick so? yay? Still mystified why it won't run on my laptop but at least that's something.

**Reply by threerings13** [reply]
I'm having this same problem.  I ran it last year on this machine.  And I wanted to update but now it just opens a split-second black window that instantly closes.  I can find python installed the same version, but IDK.  Also on Windows 10 laptop.  I tried turning off the alias thing and moving the downloaders location but nothing.
